mitodl/micromasters
# upgrade js dependencies

#### What are the relevant tickets?

none

#### What's this PR do?

I ran `yarn oudated` and 😱 

so this just upgrades a few things. I ran `yarn upgrade` to bump everything to the latest version indicated by `package.json`, and I also went through the list provided by `yarn outdated` to bump the version of a few packages up to `@latest` (instead of just the latest version satisfying the semver range we already had).

We should come back sometime soon to grab the rest of the items in the `yarn outdated` list, which are:

```
babel-plugin-transform-react-jsx 
jquery                           
keycode                          
lodash                           
material-ui                      
flow-bin                         
react-mdl                        
react-telephone-input            
sanctuary                        
sass-loader                      
searchkit                        
fetch-mock                       
sinon                            
eslint-plugin-babel              
striptags                        
chai-as-promised                 
warning                          
bootstrap                        
webpack-bundle-tracker           
react-ga                         
```

#### How should this be manually tested?

the tests should pass. I would also click around a little in the app and make sure the major features are all working ok - I tried out the learner search, profile, and profile image components,
